package.json

```diff
@@ -25,17 +25,34 @@
   "unpkg": "./dist/fuzzy-search.umd.js",
   "types": "./dist/fuzzy-search.d.ts",
   "exports": {
-    "require": "./dist/fuzzy-search.cjs",
-    "types": "./dist/fuzzy-search.d.ts",
-    "default": "./dist/fuzzy-search.modern.js"
+    ".": {
+      "module-sync": {
+        "types": "./dist/fuzzy-search.d.ts",
+        "default": "./dist/fuzzy-search.modern.js"
+      },
+      "module": {
+        "types": "./dist/fuzzy-search.d.ts",
+        "default": "./dist/fuzzy-search.modern.js"
+      },
+      "import": {
+        "types": "./dist/fuzzy-search.d.ts",
+        "default": "./dist/fuzzy-search.modern.js"
+      },
+      "default": {
+        "types": "./dist/fuzzy-search.d.cts",
+        "default": "./dist/fuzzy-search.cjs"
+      }
+    },
+    "./package.json": "./package.json"
   },
   "scripts": {
     "test": "jest --coverage",
     "dev": "microbundle watch",
-    "build": "microbundle",
+    "build": "microbundle && cp dist/fuzzy-search.d.ts ./dist/fuzzy-search.d.cts",
```

Comment:
I think this will not work on Windows. We could install cpx as a development dependency and use it instead of cp, what do you think?

Reply:
You're right it doesn't work on Windows. I fixed it in my latest commit if you wanna take a look.